image_width: 1280
image_height: 720
camera_name: head_camera
camera_matrix:
  rows: 3
  cols: 3
  data: [469.915572, 0.000000, 629.359915, 0.000000, 470.216121, 388.264688, 0.000000, 0.000000, 1.000000]
distortion_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [-0.008094, -0.014392, 0.000163, -0.000597, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1.000000, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000, 0.000000, 0.000000, 1.000000]
projection_matrix:
  rows: 3
  cols: 4
  data: [426.717255, 0.000000, 620.584588, 0.000000, 0.000000, 465.256836, 389.297820, 0.000000, 0.000000, 0.000000, 1.000000, 0.000000]
marker_101:
  rows: 1
  cols: 6
  data: [2.14, 0.1, -0.5, 1.57, 0.0, -1.57]
marker_102:
  rows: 1
  cols: 6
  data: [2.14, -0.9, -0.85, 1.57, 0.0, -1.57]
marker_103:
  rows: 1
  cols: 6
  data: [2.14, -0.1, -1.5, 1.57, 0.0, -1.57]
marker_104:
  rows: 1
  cols: 6
  data: [2.14, 0.9, -1.1, 1.57, 0.0, -1.57]
marker_105:
  rows: 1
  cols: 6
  data: [0.25, -0.98, -1.1, 1.57, 0.0, 3.141592]
marker_106:
  rows: 1
  cols: 6
  data: [-0.95, -0.98, -0.5, 1.57, 0.0, 3.141592]
marker_107:
  rows: 1
  cols: 6
  data: [-0.75, -0.98, -1.5, 1.57, 0.0, 3.141592]
marker_108:
  rows: 1
  cols: 6
  data: [1.25, -0.98, -1.5, 1.57, 0.0, 3.141592]
marker_109:
  rows: 1
  cols: 6
  data: [1.05, -0.98, -0.5, 1.57, 0.0, 3.141592]
marker_110:
  rows: 1
  cols: 6
  data: [-2.14, -0.1, -0.5, 1.57, 0.0, 1.57]
marker_111:
  rows: 1
  cols: 6
  data: [-2.14, 0.9, -0.85, 1.57, 0.0, 1.57]
marker_112:
  rows: 1
  cols: 6
  data: [-2.14, 0.1, -1.5, 1.57, 0.0, 1.57]
marker_113:
  rows: 1
  cols: 6
  data: [-2.14, -0.9, -1.1, 1.57, 0.0, 1.57]
marker_114:
  rows: 1
  cols: 6
  data: [0.05, 0.97, -1.1, 1.57, 0.0, 0.0]
marker_115:
  rows: 1
  cols: 6
  data: [1.25, 0.97, -0.5, 1.57, 0.0, 0.0]
marker_116:
  rows: 1
  cols: 6
  data: [1.05, 0.97, -1.5, 1.57, 0.0, 0.0]
marker_117:
  rows: 1
  cols: 6
  data: [-0.95, 0.97, -1.5, 1.57, 0.0, 0.0]
marker_118:
  rows: 1
  cols: 6
  data: [-0.75, 0.97, -0.5, 1.57, 0.0, 0.0]
